CMakeLists.txt

```diff
@@ -77,6 +77,13 @@ if(ENABLE_WERROR)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
 endif()
 
+if(CMAKE_CXX_COMPILER MATCHES "icpx" AND CMAKE_VERSION VERSION_LESS "3.21") 
```

Comment:
@baumeier please review these warnings?

Reply:
@junghans We can remove the `icpx` warning. The reason it did not build in parallel was the `unset(OpenMP_FOUND)` in `tools/CMakeLists.txt` for `clang`.

Reply:
I don’t understand. Wasn’t that only for versions lower than 10.

Reply:
So it said in the comment but the logic there is opposite. 
@rubengerritsen thought that there was some deeper meaning in it, as the proper OpenMP would be set somewhere else instead. But I don't know.